### DISCUSSION TITLE
I20TRELM shr_abort_abort dynpft_check_consistency mismatch

### DISCUSSION THREAD

**Discussion by pgoddard08** [E3SM-Project/E3SM#6443] [E3SM model help]
Hello,

I received the following error when attempting to run the compset I20TRELM (attached is my run bash script and the E3SM log file as text files):
From the log file:

ERROR: Unknown error submitted to shr_abort_abort.
 681:  Confirm that the year of your surface dataset
 681:  corresponds to the first year of your landuse_timeseries file
  ...
299:  dynpft_check_consistency mismatch between PCT_NAT_PATCH at initial time and tha
 299:  t obtained from surface dataset
 928:  dynpft_check_consistency mismatch between PCT_NAT_PATCH at initial time and tha
 928:  t obtained from surface dataset
 
 Thanks,
 Paul 
 
 
[e3sm_log.log](https://github.com/E3SM-Project/E3SM/files/15418291/e3sm_log.log)
[run_script_sh.txt](https://github.com/E3SM-Project/E3SM/files/15418292/run_script_sh.txt)

**Comment by jonbob** [comment]
Hi Paul,

I looked and I think you have to add this setting to user_nl_elm:
````
check_dynpft_consistency = .false.
````
Please let me know if that works?

Jon
